examples/webgpu_video_frame.html

```diff
@@ -43,14 +43,21 @@
 
 				const geometry = new THREE.PlaneGeometry();
 
-				const videoTexture = new THREE.VideoFrameTexture();
```

Comment:
I would not rename this variable since the texture is still a video texture. The shorter name makes it more readable.

Reply:
no problem